django-stubs/core/validators.pyi

```diff
@@ -65,6 +66,9 @@ def validate_integer(value: float | str | None) -> None: ...
 class EmailValidator(_Deconstructible):
     message: _StrOrPromise
     code: str
+    domain_re: str
```

Comment:
Ideally this should be converted to `ClassVar`, because there are are direct class-level usages https://github.com/django/django/blob/71a19a0e475165dbc14c1fe02f552013ee670e4c/django/core/validators.py#L212-L215

But, this can be done in a separate PR.

Reply:
Thank you, you had given me the same feedback before… but I applied it without thinking, just by looking at the codebase. I'll be more careful next time

Reply:
Thanks a lot! I highly value your contributions.